service/schema.py

```diff
@@ -0,0 +1,25 @@
+import json
+from jsonschema import validate
+
+schema = {
+    "type": "object",
+    "properties": {
+        "item_one": {"type": "string"},
+        "item_two": {"type": "number"}
+    }
+}
+
+
+def validate_schema(message_payload):
+    """
+    Validate message queue payload against json schema
+        - message_payload: queue item message
+    """
+    message_payload = json.dumps(message_payload)
+    try:
+        validate(instance=message_payload, schema=schema)
+        return True
+    except Exception as err:
+        # make sure we log errors for debugging purposes
+        # TO DO: replace with logging
+        print(err)
```

Comment:
I really, really don't like this, we're swallowing an error and just assuming (at the top level) that we know why we're getting `None` back. Better to just `raise` and handle it at the top level, so we explicitly _know_ why we didn't get back a message.